release/ray_release/reporter/db.py

```diff
@@ -1,18 +1,107 @@
 import time
+import re
 import json
 import boto3
+from typing import List
 from botocore.config import Config
 
 from ray_release.reporter.reporter import Reporter
 from ray_release.result import Result
 from ray_release.config import Test
 from ray_release.logger import logger
 
+CRASH_PATTERN_MAX_LENGTH = 4000
```

Comment:
nit: maybe move this into the `compute_crash_pattern` function, so that it is closer to its (only) usage?